quirks/shared-credentials.json

```diff
@@ -322,6 +322,15 @@
         ],
         "fromDomainsAreObsoleted": true
     },
+    {
+        "from": [
+            "flaglerclerk.com"
+        ],
+        "to": [
+            "flaglerclerk.gov"
+        ],
+        "fromDomainsAreObsoleted": false
+    },
```

Comment:
On my machine, flaglerclerk.gov appears to redirect to flaglerclerk.com, not the other way around. Am I missing something?